src/app/views/public/landing/landing.controller.coffee

```diff
@@ -4,23 +4,29 @@ angular.module 'mnoEnterpriseAngular'
 
       vm = @
       vm.isLoading = true
-
+      vm.displayAll = {label: "", active: 'active'}
+      vm.selectedCategory = vm.displayAll
       vm.appsFilter = (app) ->
-        if (vm.searchTerm? && vm.searchTerm.length > 0) || !vm.selectedCategory
+        if (vm.searchTerm? && vm.searchTerm.length > 0) || !vm.selectedCategory.label
```

Comment:
is this `.label` necessary?

Reply:
after me and @paulstuartparker took a closer look at this we realized in javascript: `!"" == true`